releases/release-1.15/release-notes-draft.md

```diff
@@ -66,7 +66,7 @@ These changes are reflected in the following Kubernetes enhancements:
 - ExecutionHook API [designed to trigger hook commands](https://github.com/kubernetes/enhancements/issues/962) in containers is now Alpha.
 - Continued deprecation of extensions/v1beta1, apps/v1beta1, and apps/v1beta2 APIs; these extensions will be retired in 1.16!
 
-Check the [release notes website](https://relnotes.k8s.io/) for the complete changelog of notable features and fixes.
+Check the [release notes website](https://relnotes.k8s.io/?releaseVersions=1.15.0) for the complete changelog of notable features and fixes.
```

Comment:
Check the [release notes website](https://relnotes.k8s.io/?releaseVersions=1.15) for the complete changelog of notable features and fixes.

Reply:
I see, compared with https://github.com/kubernetes-sigs/release-notes/pull/63 this makes totally sense. Sorry for the interruption